ruby/vampires.rb

```diff
@@ -0,0 +1,67 @@
+
+puts "How many employees will we process today? "
+employee=gets.chomp.to_i #number of employees being processed
+
+while employee>0
+	puts "What is your name? "
+	name=gets.chomp
+
+	puts "How old are you? "
+	age=gets.chomp.to_i
+
+	puts "What year were you born? "
+	year=gets.chomp.to_i
+
+	puts "Our company cafeteria serves garlic bread. Should we order some for you? (Y/N)"
+	garlic=gets.chomp.upcase
+	if garlic=="Y"
+		garlic=true
+	elsif garlic=="N"
+		garlic=false
+	else
+		puts "ERROR"
+	end
+
+	puts "Would you like to enroll in the company's health insurance? (Y/N)"
+	insurance=gets.chomp.upcase
+	if insurance=="Y"
+		insurance=true
+	elsif insurance=="N"
+		insurance=false
+	else
+		puts "ERROR"
+	end
+	#Allergy loop
+	stop=false

```

Comment:
Instead of using a variable to store a boolean and then change the boolean later in the loop, it would make sense to construct the loop to limit based on the condition you are looking for. In this case the ending condition is `answer == 'done' || answer == 'sunshine'`. Use `until answer == 'done' || answer == 'sunshine'` to make your loop more clear and reduce the extra variables.